examples/plotting/tap.py

```diff
@@ -1,3 +1,14 @@
+''' An interactive numerical tap plot based on a simple Python array of data.
+    You can select any datapoint by tapping on it. 
+    This highlights that datapoint and displays all other datapoints in a faded color.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure, bokeh.plotting.show, bokeh.models.TapTool
+    :refs: :ref:`ug_basic_tap`
```

Comment:
```suggestion
    :refs: :ref:`ug_interaction_tools_clicktap`
```
I just now checked the ref - this needs to be an actual reference from the RST documentation. This is a bit obscure sometimes, unfortunately! In this case, the reference is from `tools.rst`, linking to https://docs.bokeh.org/en/latest/docs/user_guide/interaction/tools.html#click-tap-tools